pkg/application/create_test.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/migueleliasweb/go-github-mock/src/mock"
```

Comment:
also could we add one integration test? to prove it works from outside (as blackbox), should be similar to one we already have for app create